user_templates/mcu_defaults/toolhead/LDO_Nitehawk-SB_v1.0.cfg

```diff
@@ -70,6 +70,18 @@ pin: toolhead:E_FAN
 ## and having the X endstop plugged to the toolhead MCU
 # [stepper_x]
 # endstop_pin: ^toolhead:X_STOP
+# [stepper_y]
+# endstop_pin: ^toolhead:Y_STOP
+
+## Uncomment the following two lines to configure the Nitehawk adxl sensor
+#[resonance_tester]
+#probe_points:
+## Uncomment below for 250mm build
+#    125, 125, 20
+## Uncomment below for 300mm build
+#    150, 150, 20
+## Uncomment below for 350mm build
+#    175, 175, 20
```

Comment:
Please remove this part as the resonance testing is defined elsewhere in Klippain

Reply:
done :)